### DISCUSSION TITLE
1주일간 해보니까?

### DISCUSSION THREAD

**Discussion by kyupid** [spring-god-study/spring-basic#12] [General]
자유롭게 개선할점 의견 제시부탁드립니다

**Comment by dnjsrud3407** [comment]
확실히 같이하는 분이 계시니까 작심삼일 안하고 꾸준히 하게 되는거같아요😊
제가 밤에 일찍 자서 그런데 이슈를 밤 9시 이전에는 올려주셨으면 좋겠습니다~ㅎㅎ
